apps/src/music/player/MusicPlayer.js

```diff
@@ -187,39 +187,39 @@ export default class MusicPlayer {
     return this.soundEvents;
   }
 
+  /*
   getCurrentAudioElapsedTime() {
     if (!this.isPlaying) {
       return 0;
     }
 
     return GetCurrentAudioTime() - this.startPlayingAudioTime;
   }
+  */
 
-  getPlayheadPosition() {
-    if (!this.isPlaying) {
-      return 0;
-    }
-
-    // Playhead time is 1-based (user-facing)
-    return 1 + this.getCurrentAudioElapsedTime() / this.secondsPerMeasure();
-  }
-
-  getCurrentMeasure() {
+  // Called by generated code in the interpreter, and returns the current
+  // playhead, in floating point for an exact position, and scaled to measures.
+  // It's 1-based.
+  // Returns 0 if music is not playing.
+  getCurrentPlayhead() {
```

Comment:
nit: `getCurrentPlayheadPosition` or `playheadTime`? Though using `time` here might get back to being confusing 1-based time with 0-based time

Reply:
Good question.  It's actually scaled to measures, so `time` is confusing in that regard as well.  :). 
`getCurrentPlayheadPosition` seems good though.